elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java

```diff
@@ -830,9 +830,10 @@ public String getNameFromAlias(Object entity, String alias) {
      * @param entityClass the entity type to check for the shareable permissions
      * @return true if entityClass is shareable.  False otherwise.
      */
-    public boolean isShareable(Class<?> entityClass) {
-        return getAnnotation(entityClass, SharePermission.class) != null
-                && getAnnotation(entityClass, SharePermission.class).sharable();
+    public boolean isTransferable(Class<?> entityClass) {
+        NonTransferable nonTransferable = getAnnotation(entityClass, NonTransferable.class);
+
+        return (nonTransferable == null || nonTransferable.enabled() == false);
```

Comment:
` == false` looks strange.  Can we say
```suggestion
        return (nonTransferable == null || ! nonTransferable.enabled());
```